test/bayes/bayesian_common_benchmarks.rb

```diff
@@ -1,9 +1,28 @@
 # encoding: utf-8
 
 module BayesianCommonBenchmarks
-  def load_data
-    sms_spam_collection = File.expand_path(File.dirname(__FILE__) + '/../data/corpus/SMSSpamCollection.tsv')
-    File.read(sms_spam_collection).force_encoding("utf-8").split("\n")
+  MAX_RECORDS = 5000
+
+  class BenchmarkReporter < Minitest::Reporters::RubyMateReporter
```

Comment:
I would have much preferred to extract this custom reporter into a separate file and potentially share that with other benchmarks in future, but the kind of data I am using in the `before_suite` method will not be available outside, unless there is a way that I am unaware of.